### PR TITLE
Add function for printing SDP attributes

### PR DESCRIPTION
**Pull request by wosrediinanatour**
It adds a function `pjmedia_sdp_attr_print()` similar to `pjmedia_sdp_media_print()`.

### CODE REVIEW

File: pjmedia/include/pjmedia/sdp.h [hunk 1]
```diff
@@ -508,6 +508,18 @@ PJ_DECL(pjmedia_sdp_media*)
 pjmedia_sdp_media_clone( pj_pool_t *pool, 
                          const pjmedia_sdp_media *rhs);
 
+/**
+ * Print attribute to a buffer.
+ *
+ * @param media     The attribute.
```
Comment (nanangizz) [inline pjmedia/include/pjmedia/sdp.h:514]: should be `attr` instead of `media`?

File: pjmedia/include/pjmedia/sdp.h [hunk 2]
```diff
@@ -489,12 +489,12 @@ typedef struct pjmedia_sdp_media pjmedia_sdp_media;
  *
  * @param media     The media description.
  * @param buf       The buffer.
- * @param size      The buffer length.
+ * @param legth     The buffer length.
```
Comment (nanangizz) [inline pjmedia/include/pjmedia/sdp.h:492]: typo: `legth`

**Review by nanangizz — COMMENTED** [review]
Thanks for the PR.
Could you also describe a sample scenario that this function is needed by app?

File: pjmedia/include/pjmedia/sdp.h (hunk 2 above)
Comment (sauwming) [reply pjmedia/include/pjmedia/sdp.h:492]: I'm not sure why you need to change the parameter name here.

IMO, it's better to leave existing API's parameter names untouched, unless there's a reason for the change.

**Comment by wosrediinanatour** [comment]
> Could you also describe a sample scenario that this function is needed by app?

There are already exposed functions for printing SDP (`pjmedia_sdp_print()`) and the SDP's media description (`pjmedia_sdp_media_print()`). For C++ wrappers that wrap the types `pjmedia_sdp_session`, `pjmedia_sdp_media`, and `pjmedia_sdp_attr`, it is handy to have consistent functionality to print the content (even if it is for logging "only") into a string (buffer).

**Review by sauwming — APPROVED** [review]

**Review by trengginas — APPROVED** [review]

**Review by nanangizz — APPROVED** [review]

**Pull request merged**